src/cloudant/_common_util.py

```diff
@@ -263,6 +263,20 @@ def get_docs(r_session, url, encoder=None, headers=None, **params):
     resp.raise_for_status()
     return resp
 
+#pylint: disable=unused-argument
+def append_response_error_content(response, **kwargs):
+    """
+    Provides a helper to add HTTP response error and reason messages.
+    """
+    try:
+        if (response.status_code >= 400 and response.json() and
+                ('error' in response.json()) and ('reason' in response.json())):
```

Comment:
I'd prefer to see the checks broken out because I believe there are cases where we get only one of the fields. In those cases it would be good to print what we have without bypassing because the other field is not present.